enacts/wat_bal/maproom_monit.py

```diff
@@ -134,9 +136,13 @@ def update_time_sel(planting_day, planting_month, graph_click, current_options):
         ).squeeze()
         time_range = time_range.where(
             time_range >= p_d, drop=True
-        ).dt.strftime("%-d %b %y").values
-        the_value = time_range[-1]
-    return time_range, the_value
+        )
+        time_options = [
+            {"label": tr.dt.strftime("%-d %b %y").values, "value": tr.dt.strftime("%Y-%m-%d").values}
```

Comment:
for the label, %-d means the leading 0 is not printed and that is what is shown to the user, e.g. 8 Jan 2009, as a somewhat human friendly way of printing dates. For the values, it creates e.g. 2009-01-08, a string format that is readily understood by Xarray and friends and used as very first examples in Xarray [doc](https://docs.xarray.dev/en/stable/user-guide/time-series.html).

So that when time_selection is an input of a callback, it can be used readily in slice or what have you.

I guess in more recent work for flex. fcst. Maproom, we've been cleaner on what we give to and take from menus that propose dates, so maybe when that piece is merged there (maybe it is already), I can make an Issue to port (and maybe further clean-up / generalize) this handling of dates in menus and callbacks to other Maprooms, including this one here?